dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -354,6 +354,7 @@ BOOL WINAPI PickIconDlg(
     UINT nMaxFile,
     INT* lpdwIconIndex)
 {
+    CCoInit ComInit; // For SHAutoComplete [CORE-20030]
```

Comment:
We usually employ `(CORE-xxx)` in the code rather than the square brackets.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%28CORE-
vs.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%5BCORE-